cloud_experts_tutorials/rosa-mobb-dynamic-certificate-custom-domain-tutorial.adoc

```diff
@@ -0,0 +1,531 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-dynamic-certificate-custom-domain-tutorial"]
+= Tutorial: Dynamically issuing certificates with a custom domain in ROSA
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-dynamic-certificate-custom-domain-tutorial
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-20
+//---
+//date: '2022-10-11'
+//title: Dynamic Certificates for ROSA Custom Domain
+//weight: 1
+//tags: ["AWS", "ROSA"]
+//authors:
+//   Kevin Collins
+//---
+
+There may be situations when you prefer not to use `wild-card` certificates. This Red Hat OpenShift Service on AWS (ROSA) guide talks about certificate management with cert-manager and letsencrypt, to dynamically issue certificates to routes created on a custom domain that's hosted on AWS Route53.
+
+== Prerequisites
+
+* A ROSA cluster
+* You are logged into the `oc` CLI
+* You are logged into the AWS CLI
+* You are logged into the `rosa` CLI
+* jq
+* gettext
+* A public Route53 hosted zone, and the related domain to use
+
+== Set up environment
+
+. Export a few environment variables:
++
+[source,terminal]
+----
+   export CLUSTER_NAME="sts-pvtlnk-cluster"
+   export OIDC_PROVIDER=$(oc get authentication.config.openshift.io cluster -o json \
+   | jq -r .spec.serviceAccountIssuer| sed -e "s/^https:\/\///")
+   export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+   export SCRATCH_DIR=/tmp/scratch
+   export AWS_PAGER=""
+   export LETSENCRYPT_EMAIL=youremail@work.com
+   export HOSTED_ZONE_ID=ABCDEFGHEXAMPLE
+   export HOSTED_ZONE_REGION=us-east-2
+   export DOMAIN=lab.domain.com   #Custom Hosted Zone Domain for apps
+   mkdir -p $SCRATCH_DIR
+----
+
+. Install jq & gettext. Installing or ensuring the gettext & jq package is installed allows you to use envsubst to simplify some of your configuration. You can then use the output of the CLI commands as the input into YAML files to reduce the complexity of manual editing.
```

Comment:
Probably a good idea to use "and" rather than an ampersand here, and to put everything after the initial instruction in a subsequent paragraph to make the step clearer.